### DISCUSSION TITLE
build failed: cannot find symbol

### DISCUSSION THREAD

**Discussion by MaliaCataleya** [eclipse-edc/Connector#2480] [Q&A]
Hey!

Unfortunately I'm experiencing an error when trying to build the repo.

In various files 'var' seems to not be recognized - e.g.:
TestSummaryPlugin.java:33: error: cannot find symbol
symbol:   class var
location: class org.eclipse.dataspaceconnector.gradle.TestSummaryPlugin.AfterSuitePrinter

Can anyone help me out?

Thanks in advance!

**Comment by wolf4ood** [comment]
Hi @MaliaCataleya 

Which version of Jdk are you using?

Make sure to have at least jdk 11 installed on your machine

Thanks

**Reply by MaliaCataleya** [reply]
Hi @wolf4ood 

I am using jdk 11.

The output when running: java -version

java version "11.0.17" 2022-10-18 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.17+10-LTS-269)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.17+10-LTS-269, mixed mode)

**Comment by ndr-brt** [comment]
> In various files 'var' seems to not be recognized - e.g.: TestSummaryPlugin.java:33: error: cannot find symbol symbol: class var location: class org.eclipse.dataspaceconnector.gradle.TestSummaryPlugin.AfterSuitePrinter

`TestSummaryPlugin` has been moved to another repository (GradlePlugins) some times ago, and the `org.eclipse.dataspaceconnector` is the old one.
Please checkout the current `main` branch and try it again with `./gradlew clean build`
